pjsip/src/pjsua-lib/pjsua_core.c

```diff
@@ -3817,6 +3817,15 @@ static void restart_listener_cb(void *user_data)
 }
 
 
+static void ip_change_put_back_inv_config(void *user_data)
+{
+    PJ_UNUSED_ARG(user_data);
+
+    PJ_LOG(4,(THIS_FILE,"IP change stops ignoring request timeout"));
+    pjsip_cfg()->endpt.keep_inv_after_tsx_timeout = PJ_FALSE;
```

Comment:
Shouldn't we revert it to the previous value instead?

Reply:
The timer is scheduled only when the value is `PJ_FALSE`.